packages/hash/frontend/package.json

```diff
@@ -21,17 +21,12 @@
     "@blockprotocol/hook": "0.0.8",
     "@blockprotocol/type-system": "0.0.1",
     "@dnd-kit/core": "6.0.5",
-    "@dnd-kit/modifiers": "6.0.0",
     "@dnd-kit/sortable": "7.0.1",
     "@dnd-kit/utilities": "3.2.0",
-    "@emoji-mart/data": "1.0.5",
```

Comment:
I don’t see any references to `@emoji-mart/data` except in `README.md`. I think that the library downloads the list from the CDN if the prop is not provided. So removing `@emoji-mart/data` should not break or change anything.

<img width="867" alt="Screenshot 2022-12-09 at 14 15 47" src="https://user-images.githubusercontent.com/608862/206722176-08702684-b75c-490c-908e-e3187ab43608.png">

However, if we want, we can schedule a new task and replace CDN data with local data. This will increase bundle size though.